guides/common/modules/proc_puppet-overriding-puppet-parameters-location-based.adoc

```diff
@@ -0,0 +1,18 @@
+[id="puppet_guide_overriding_puppet_parameters_location_based"]
+= Overriding Puppet Parameters Location Based
+
+You can use groups of hosts to override Puppet parameters for multiple hosts at once.
+The following examples chooses the _location_ context to illustrate setting context based parameters.
+
+.Procedure
+. Navigate to *Configure > Classes* and select the *Smart Class Parameter* tab.
```

Comment:
```suggestion
. In the {Project} web UI, navigate to *Configure > Classes* and select the *Smart Class Parameter* tab.
```

Reply:
Fixed using the new `ProjectWebUI` attribute.